assignment-client/src/entities/EntityTreeSendThread.h

```diff
@@ -40,6 +42,7 @@ class EntityTreeSendThread : public OctreeSendThread {
 
     DiffTraversal _traversal;
     EntityPriorityQueue _sendQueue;
+    std::unordered_set<EntityItemPointer> _entitiesToSend;
```

Comment:
`_entitiesToSend` keeps a smart-pointer to the `EntityItem`, which means it will prevent the entity from being deleted on another thread.  Meanwhile the `_sendQueue` ultimately keeps a weak-pointer (inside the `PrioritizedEntit`y struct) to prevent just such a situation.  I'd like to make this `unordered_set` also keep a weak-pointer or something, but it isn't clear to me exactly what it should be.

The value in `_entitiesToSend` is never actually used.  It is just a record with faster lookup of what is in the `_sendQueue`.  Which means maybe we could just use the raw pointer.  However I wonder what would be the likelihood  for an `EntityItem*` to be deleted and then a brand new `EntityItem` instance created at the same location?  BTW, are weak-pointers properly invalidated when such an address recycle happens, or do they magically point to the new instance?  I dunno.